src/main.cpp

```diff
@@ -1345,13 +1347,27 @@ void wiFiSetup() {
     wm.setConnectTimeout(10); // Try 10 sec to connect to WLAN, 5 sec too short!
     wm.setBreakAfterConfig(true);
     wm.setConnectRetries(3);
-    //wm.setWiFiAutoReconnect(true);
+    //wm.setWiFiAutoReconnet(true);
+    // Wifisetup fro
+    sysParaWifiSetup.getStorage() ;
+    Serial.println("XXXXXXXX");
+    Serial.println(WifiSetup);
+    if(WifiSetup == 0)
+    {
+            
+            const char hostname[] = (STR(HOSTNAME)) ;
+            displayLogo("connect to  AP: ",HOSTNAME);
```

Comment:
```suggestion
            displayLogo("Connect to Wifi: ", HOSTNAME);
```

Reply:
should also use translation strings though

Reply:
Implemented in the last commit